File: uri/1st Page/cp with c/URI 1010.c
#include<stdio.h>
int main()
{
    int cp1, cp2, p1, p2;
    double pr1, pr2, paid;
    scanf("%d %d %lf", &cp1, &p1, &pr1);
    scanf("%d %d %lf", &cp2, &p2, &pr2);
    paid = (p1*pr1)+(p2*pr2);
    printf("VALOR A PAGAR: R$ %.2lf\n",paid);
    return 0;
}

File: uri/1st Page/cp with java/src/com/URI1010.java
import java.util.Scanner;
public class URI1010 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int cd1 = sc.nextInt();
        int unt1 = sc.nextInt();
        double prc1 = sc.nextDouble();
        int cd2 = sc.nextInt();
        int unt2 = sc.nextInt();
        double prc2 = sc.nextDouble();
        double pr1 = unt1 * prc1;
        double pr2 = unt2 * prc2;
        double total = pr1 + pr2;
        System.out.printf("VALOR A PAGAR: R$ %.2f\n",total);
    }
}
